internal/transactions/send.go

```diff
@@ -69,19 +73,31 @@ func send(
 		return nil, fmt.Errorf("error loading transaction file: %w", err)
 	}
 
-	txArgs, err := flowkit.ParseArguments(sendFlags.Arg, sendFlags.ArgsJSON)
+	codes := map[common.LocationID]string{}
+	location := common.StringLocation(codeFilename)
+	program, must := cmd.PrepareProgram(string(code), location, codes)
+	checker, _ := cmd.PrepareChecker(nil, location, codes, nil, must)
```

Comment:
Would it make sense if this was moved to `ParseArgumentsWithoutType`? If you have a preference feel free to leave it here.